distribution/docs/src/main/resources/content/_integrating/_endpoints/crud-endpoints-intro.adoc

```diff
@@ -0,0 +1,15 @@
+:type: endpointIntro
+:status: published
+:title: CRUD Endpoints
+:operations: crud
+:order: 02
+
+== {title}
+
+To perform CRUD (Create, Read, Update, Delete) operations on data or metadata in the catalog, work with one of these endpoints.
+
+[NOTE]
+====
+Not all installations will expose all available endpoints. Check with ${branding} administrator to confirm availability of these endpoints.
+====
```

Comment:
❓ Could this NOTE be put in the summary of the overarching "Endpoints" section instead of being repeated in each subsection?

Reply:
certainly